core/src/main/java/de/danoeh/antennapod/core/util/ChapterUtils.java

```diff
@@ -65,10 +72,18 @@ public static void loadChapters(Playable playable, Context context) {
             if (feedMedia.getItem().hasChapters()) {
                 chaptersFromDatabase = DBReader.loadChaptersOfFeedItem(feedMedia.getItem());
             }
+
+            if (! StringUtil.isBlank(feedMedia.getItem().getChapterUrl())) {
```

Comment:
From the StringUtil documentation:
> A minimal String utility class. Designed for internal jsoup use only - the API and outcome may change without notice. 

I would use Android's `TextUtils.isEmpty`